### PR TITLE
[4.0] xml review

### PR DESCRIPTION
**Pull request by brianteeman**
converts spaces to tabs.

Code review required

### CODE REVIEW

**Comment by chmst** [comment]
Wouldn't it be a good time to delete als all the size="xx" attributes? They come from former versions and asmuch as I know, are not used in J4. But not in this PR, of course

**Comment by brianteeman** [comment]
For another PR of course

File: components/com_newsfeeds/tmpl/newsfeed/default.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	<!-- Add fields to the request variables for the layout. -->
 	<fields name="request">
 		<fieldset name="request"
-	            addfieldprefix="Joomla\Component\Newsfeeds\Administrator\Field"
+				addfieldprefix="Joomla\Component\Newsfeeds\Administrator\Field"
```
Comment (richard67) [inline components/com_newsfeeds/tmpl/newsfeed/default.xml]: Isn’t that too much indent?

**Comment by richard67** [comment]
To me it seems to be too many tabs at several places.

**Comment by brianteeman** [comment]
i fixed newsfeeds - don't see any other issues

File: build/phpmd.xml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-                     http://pmd.sf.net/ruleset_xml_schema.xsd"
-         xsi:noNamespaceSchemaLocation="
-                     http://pmd.sf.net/ruleset_xml_schema.xsd">
-    <description>
-        Custom rule set for the Joomla CMS.
-    </description>
+		 xmlns="http://pmd.sf.net/ruleset/1.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+					 http://pmd.sf.net/ruleset_xml_schema.xsd"
```
Comment (richard67) [inline build/phpmd.xml]: Here

File: build/phpmd.xml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+		 xsi:noNamespaceSchemaLocation="
+					 http://pmd.sf.net/ruleset_xml_schema.xsd">
```
Comment (richard67) [inline build/phpmd.xml]: And here

File: build/phpmd.xml (hunk 2 above)
Comment (brianteeman) [reply build/phpmd.xml]: it looks like this file is using a different standard as the tabs are also used for column alignment.

all this pr does is change the spaces to tabs.

if this one file is a problem for you I would rather remove the file from the PR than waste more time

File: build/phpmd.xml (hunk 3 above)
Comment (brianteeman) [reply build/phpmd.xml]: it looks like this file is using a different standard as the tabs are also used for column alignment.

all this pr does is change the spaces to tabs.

if this one file is a problem for you I would rather remove the file from the PR than waste more time

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3a8b4829f92bbedbfd205e73ca856ff8b0dbe4d3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35700">issues.joomla.org/tracker/joomla-cms/35700</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 3a8b4829f92bbedbfd205e73ca856ff8b0dbe4d3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35700">issues.joomla.org/tracker/joomla-cms/35700</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35700">issues.joomla.org/tracker/joomla-cms/35700</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**